src/acacia/settings.py

```diff
@@ -86,22 +88,22 @@
 WSGI_APPLICATION = 'acacia.wsgi.application'
 
 DATABASES = {
-    'default': {
+    'postgres': {
```

Comment:
```suggestion
    'default': {
```
Faz mais sentido deixar esse como default por conta de como o deploy tá sendo desenvolvido.